helpers/lib/resourceHints.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable curly */
+const _ = require("lodash");
```

Comment:
Please, don't use `lodash`, we have a ticket in backlog to remove it from dependencies